### ISSUE TITLE
[Marker.io] Team Block missing keyboard focus outline

### ISSUE THREAD

**Issue by christophermhinds** [equalizedigital/student-loan-planner-theme#435]
[![Screenshot_2023-10-19_21-49-46.png](https://mrkr.io/s/6531a47a24bf4e04a7d68aa2/2)](https://mrkr.io/s/6531a47a24bf4e04a7d68aa2/0)

Not getting an outline on these blocks when tabbing through them.

---

**Reported by:** equalizedigital (dev@equalizedigital.com) - [Contact via Marker.io](https://app.marker.io/i/6531a47a24bf4e04a7d68aa5_a4934a76364d2e15?advanced=1&comments=1)

**Source URL:** [https://studentloanprd.wpenginepowered.com/about-us/team/](https://studentloanprd.wpenginepowered.com/about-us/team/)

**Issue details:** [Open in Marker.io](https://app.marker.io/i/6531a47a24bf4e04a7d68aa5_a4934a76364d2e15?advanced=1)

**Console:** [📃  0 logs](https://app.marker.io/i/6531a47a24bf4e04a7d68aa5_a4934a76364d2e15?advanced=1&activePane=console)

**Network:** [View network logs](https://app.marker.io/i/6531a47a24bf4e04a7d68aa5_a4934a76364d2e15?advanced=1&activePane=console)

**Session replay:** [Watch last 30 seconds](https://app.marker.io/i/6531a47a24bf4e04a7d68aa5_a4934a76364d2e15?advanced=1&sessionReplay=1)

<table><tr><td><strong>Device type</strong></td><td>desktop</td></tr><tr><td><strong>Browser</strong></td><td>Chrome 116.0.0.0</td></tr><tr><td><strong>Screen Size</strong></td><td>1397 x 992</td></tr><tr><td><strong>OS</strong></td><td>OS X 13.5.1</td></tr><tr><td><strong>Viewport Size</strong></td><td>1397 x 988</td></tr><tr><td><strong>Pixel Ratio</strong></td><td>@&#8203;2x</td></tr></table>
